### ISSUE TITLE
Cannot find module '../build/Debug/nodegit.node'

### ISSUE THREAD

**Issue by danielcrisp** [tad-lispy/npm-git-install#5]
Hi, I'm getting the following error when using your module.

Any ideas?

``` shell
> npm-git-install

module.js:442
    throw err;
    ^

Error: Cannot find module '../build/Debug/nodegit.node'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/xxx/node_modules/nodegit/lib/nodegit.js:17:12)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)

npm ERR! Darwin 15.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "i"
npm ERR! node v6.2.1
npm ERR! npm  v3.9.3
npm ERR! code ELIFECYCLE
npm ERR! angular2-seed@0.0.0 install: `npm-git-install`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the angular2-seed@0.0.0 install script 'npm-git-install'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the angular2-seed package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm-git-install
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs angular2-seed
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls angular2-seed
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /xxx/npm-debug.log
```

I've tried node v5.6.0 and v6.2.1

Let me know if you want to see the npm-debug.log

**Comment by danielcrisp** [comment]
Ok, running `npm install` inside your module's directory fixed it. e.g. 

```
cd node_modules/npm-git-install
npm install
# return to project root
cd ../../
npm install
```

Is that the correct way to do it?

**Comment by tad-lispy** [comment]
Hello @danielcrisp. Thanks for reporting and glad you have found a workaround. It should however just work without extra steps. 

Did you install `npm-git-install` from NPM (i.e. `npm install --save npm-git-install`) or from GitHub (with `git clone` or something similar)?

**Comment by danielcrisp** [comment]
I installed it inside my project using `npm install --save npm-git-install`

**Comment by tad-lispy** [comment]
Should be fixed in #9. Would you please follow the instructions in this PR and let me know if it works for you? Thank you very much.
